Classes/Domain/Search/ResultSet/Facets/OptionBased/Options/OptionsPackage.php

```diff
@@ -17,7 +17,7 @@
 use ApacheSolrForTypo3\Solr\Domain\Search\ResultSet\Facets\AbstractFacetPackage;
 
 /**
- * Class HierarchyPackage
+ * Class OptionsPackage
  * @package ApacheSolrForTypo3\Solr\Domain\Search\ResultSet\Facets\OptionBased\Options
```

Comment:
`@package *` can be removed as well.

Reply:
I added a commit which removes all @package references from the class docblock.